fix: Rephrasing and improved output for framework quick starts instructions (#47576)
supabase/supabase: apps/docs/content/_partials/quickstart_db_setup.mdx

## 2. Set up your database

When your Supabase project is up and running, create an `instruments` table with some sample data. Then set only the privileges each Postgres role needs, add [Row Level Security (RLS)](/docs/guides/database/postgres/row-level-security) for enhanced security for database data by default, and create an RLS policy to make the data in the table publicly readable.

<Admonition type="tip">

> Save some steps by <a href={`/dashboard/project/_/sql/new?content=${encodeURIComponent(sqlSetup)}`}>clicking here to prefill the SQL</a> and execute it.

Reviewer: ```suggestion
Save some steps by <a href={`/dashboard/project/_/sql/new?content=${encodeURIComponent(sqlSetup)}`}>clicking here to prefill the SQL</a> in the SQL Editor, and then clicking **Run**.
```

Clarifying what you still need to do, which it makes it sound less appealing…

</Admonition>

> Or do this manually by going into your project's [SQL Editor](/dashboard/project/_/sql/new) and click **Run**.

Reviewer: ```suggestion
Or do this manually in our project's [SQL Editor](/dashboard/project/_/sql/new), by pasting the SQL below, and clicking **Run**.
```
